src/Okta.Idx.Sdk/AuthenticationStatus.cs

```diff
@@ -0,0 +1,27 @@
+﻿// <copyright file="AuthenticationStatus.cs" company="Okta, Inc">
+// Copyright (c) 2020 - present Okta, Inc. All rights reserved.
+// Licensed under the Apache 2.0 license. See the LICENSE file in the project root for full license information.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Okta.Idx.Sdk
+{
+    /// <summary>
+    /// This enum represents all the possible authentication response statuses
+    /// </summary>
+    public enum AuthenticationStatus
+    {
+        /// <summary>
```

Comment:
We should consider adding a `FAILURE` status also which would be set in case of any backend issues or even exceptions.

Reply:
When failures or backend issues happen I throw an exception.